src/main/java/net/snowflake/client/jdbc/cloud/storage/S3HttpUtil.java

```diff
@@ -30,11 +35,19 @@ public static void setProxyForS3(HttpClientSettingsKey key, ClientConfiguration
       clientConfig.setProxyHost(key.getProxyHost());
       clientConfig.setProxyPort(key.getProxyPort());
       clientConfig.setNonProxyHosts(key.getNonProxyHosts());
+      StringBuilder logBuilder = new StringBuilder(
+              String.format("Setting S3 proxy. Host: %s, port: %d, protocol: %s, non-proxy hosts: %s",
```

Comment:
let's use builder or format - we should not mix them (also in other places)

Reply:
Changed it.